eng/testing/linker/project.csproj.template

```diff
@@ -2,9 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+    <MonoAOTCompilerDir>{MonoAOTCompilerDir}</MonoAOTCompilerDir>
+    <MonoProjectRoot>{MonoProjectRoot}</MonoProjectRoot>
+    <EventSourceSupport>true</EventSourceSupport>
```

Comment:
This should be deleted now that we have

```<UseDefaultBlazorWASMFeatureSwitches>false</UseDefaultBlazorWASMFeatureSwitches>```

right?

Reply:
ohh good catch, yes you are correct, I'll remove that.